anti-analysis/anti-av/check-for-sandbox-and-av-modules.yml

```diff
@@ -3,8 +3,9 @@ rule:
     name: check for sandbox and av modules
     namespace: anti-analysis/anti-av
     authors:
-      - "@_re_fox"
-    scope: basic block
+      - '@_re_fox'
+    scopes:
+      static: basic block
```

Comment:
this rule could be run on `dynamic: thread`?

Reply:
hmm, perhaps it should be `dynamic: process` instead?

Reply:
or that, yes

Reply:
i think we should use the most specific scope as possible. so we should prefer thread rather than process.

im guessing that we'll want to use thread in almost all cases, unless we're looking for combinations of behaviors across an entire program, for which we only have a few rules today.

thoughts?

Reply:
furthermore, in this specific example, i think this should be `call`, because we're trying to match `GetModuleHandle("avghook.dll" or "snxhk.dll" or...)`.

im not sure if automated script can help us here. maybe basic block should map to call if feasible?

Reply:
or at least flag for manual review